Sources/ATProtoKit/Utilities/ATFacetParser.swift

```diff
@@ -95,6 +95,60 @@ public class ATFacetParser {
 
         return spans
     }
+    
+    /// Replaces URL text in the original string with a truncated version used for display in Bluesky.
```

Comment:
```suggestion
    /// Replaces URL text in the original string with a truncated version used for display in Bluesky.
    ///
```

For the sake of formatting, I just want to add one line.